doc/source/reshaping.rst

```diff
@@ -801,3 +801,40 @@ Note to subdivide over multiple columns we can pass in a list to the
 
    df.pivot_table(
        values=['val0'], index='row', columns=['item', 'col'], aggfunc=['mean'])
+
+.. _reshaping.explode:
+
+Exploding a List-like Column
+----------------------------
+
+.. ipython:: python
+   :suppress:
```

Comment:
Maybe don't suppress this? I think users may want to see the input.

Reply:
Makes sense. It'll be easier for them to just copy/paste into a repl to try it out.